pints/_log_likelihoods.py

```diff
@@ -237,6 +237,240 @@ def __call__(self, x):
         )
 
 
+class CombinedGaussianLogLikelihood(pints.ProblemLogLikelihood):
```

Comment:
Just wondering about the name? Is this what this type of model is called in the literature or is it one we've made up? To me, it isn't that informative about the type of noise model (but that's perhaps because I'm not au fait with the literature). I would call it something like `MultiplicativeAndConstantGaussianLogLikelihood`.

Reply:
In the PKPD community "combined Gaussian error" seems to be quite popular. I personally don't really mind the name too much, but `pints.MultiplicativeAndConstantGaussianLogLikelihood()` has already 52 characters. So simply from a flake8 perspective this name may be a little too long.